libraries/nexus_helper.rb

```diff
@@ -37,5 +37,13 @@ def wait_until_ready!(endpoint, timeout = 15 * 60)
     rescue Timeout::Error
       raise Nexus3NotReady.new(endpoint, nexus_timeout)
     end
+
+    # Load Groovy scripts via the Nexus3 API.
+    class Scripting
+      def self.groovy_script_location(script_name, node)
+        @cookbook ||= node.run_context.cookbook_collection['nexus3']
```

Comment:
do you really need to memoize the cookbook?

Reply:
Not sure. I thought this would help but it is one lookup in a hash anyway, and the class is likely not reused (maybe it is?)